src/Frontend/Themes/Bootstrap/src/Layout/Templates/Base.html.twig

```diff
@@ -34,13 +34,11 @@
 
   {% block icons %}
     {# Apple touch icon #}
-    {# @todo create the below apple touch icon files #}
```

Comment:
Gaat dat dan niet vergeten worden?

Reply:
Dat staat in de slice checklist en ik maak er tegenwoordig ook een aparte taak voor aan. Ik heb een workflow ding om die allemaal automatisch aan te maken.
Als je dat veiliger vind laat ik de comments staan en zet ik in de template taak om de todo comments te verwijderen als de favicons gegenereerd zijn.